mace/calculators/foundations_models.py

```diff
@@ -32,7 +32,8 @@
     "mace-matpes-pbe-0": "https://github.com/ACEsuit/mace-foundations/releases/download/mace_matpes_0/MACE-matpes-pbe-omat-ft.model",
     "mace-matpes-r2scan-0": "https://github.com/ACEsuit/mace-foundations/releases/download/mace_matpes_0/MACE-matpes-r2scan-omat-ft.model",
 }
-mace_mp_names = [None] + list(mace_mp_urls.keys())
+mace_mp_constants = list(mace_mp_urls.keys())
```

Comment:
What is "constants" supposed to convey here?  And what is mace_mp_names actually used for?

Reply:
ok was a reason for it but i fixed it using filter None since was used in only one place.